### PR TITLE
targetSafetyMarginSec support

### PR DESCRIPTION
**Pull request by aburchell**
After the stimulus is done being presented, delay presenting the alphabet and response instructions.

### CODE REVIEW

**Comment by aburchell** [comment]
@peilingjiang Should I be worried about failing all the Netlify checks?

**Comment by peilingjiang** [comment]
> @peilingjiang Should I be worried about failing all the Netlify checks?

Normally, yes, let me take a quick look at why.

**Comment by peilingjiang** [comment]
I think it's nothing to do with your code. The building is failing because it fails to build the example, due to the `targetAlphabet` row that hasn't been updated. Can you update that for this PR? Or we can do it in the main after merging.

File: threshold.js [hunk 1]
```diff
@@ -2021,7 +2023,7 @@ const experiment = (blockCount) => {
       }
       // update/draw components on each frame
 
-      const uniDelay = 0; // 0.5 by default?
+      const uniDelay = 0;
```
Comment (peilingjiang) [inline threshold.js:2026]: I think the `uniDelay` is for the same purpose? Maybe we can pass the `targetSafetyMarginSec` value into it? Or we can rename/delete it at all.

Comment (aburchell) [reply threshold.js:2026]: No, `uniDelay` comes before the stimulus is presented. `targetSafetyMarginSec` comes after the stimulus is undrawn, before the alphabet and response-instructions show.

Comment (peilingjiang) [reply threshold.js:2026]: Oh I see, then it's fine to leave uniDelay there

**Comment by aburchell** [comment]
> I think it's nothing to do with your code. The building is failing because it fails to build the example, due to the `targetAlphabet` row that hasn't been updated. Can you update that for this PR? Or we can do it in the main after merging.

Ah, yes I see. I had made that change while working with Denis on the new bounding logic, but those changes aren't tested enough to push. I'll make a new PR for the `alphabet` to `characterSet` change on `main` then? And merge those changes into here before resolving this PR?

**Comment by peilingjiang** [comment]
I think either way works, the only thing causing this failure is the Glossary has been updated. But it seems not a good idea to just roll it back since there are useful updates for margin sec etc. I would say we just merge and let it fail for now.

**Pull request merged**